packages/transport/src/lowlevel/sharedConnectionWorker.ts

```diff
@@ -6,9 +6,8 @@
 // about intent to acquire/release and then send another message when that is done.
 // Other windows then can acquire/release
 
-// @ts-ignore
-import { create as createDefered } from '../utils/defered';
-import type { Deferred } from '../utils/defered';
+import { createDeferred } from '@trezor/utils';
```

Comment:
```suggestion
import { createDeferred, Deferred } from '@trezor/utils';
```
like everywhere else, i've missed that before

Reply:
done